dask/array/tests/test_reductions.py

```diff
@@ -22,6 +22,10 @@ def assert_eq(a, b):
 def test_numel(dtype, keepdims):
     x = np.ones((2, 3, 4))
 
+    for length in range(x.ndim):
+        assert_eq(da.reductions.numel(x, axis=0, keepdims=keepdims, dtype=dtype),
```

Comment:
Are you sure this is right? `length` isn't used anywhere in this loop.

Reply:
Right, thanks for noticing that. Did it in a hurry. :)